docs/docsite/rst/dev_guide/module_lifecycle.rst

```diff
@@ -1,50 +1,126 @@
 .. _module_lifecycle:
 
-**********************************
-The lifecycle of an Ansible module
-**********************************
+********************************************
+The lifecycle of an Ansible module or plugin
+********************************************
 
-Modules in the main Ansible repo have a defined life cycle, from first introduction to final removal. The module life cycle is tied to the `Ansible release cycle <release_cycle>`.
-A module may move through these four states:
+Modules and plugins in the main Ansible repository have a defined life cycle, from first introduction to final removal. The module and plugin life cycle is tied to the `Ansible release cycle <release_cycle>`.
+A module or plugin may move through these four states:
 
-1. When a module is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
+1. When a module or plugin is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
 
-2. If a module matures, we will remove the 'preview' mark in the documentation. We support (though we cannot guarantee) backwards compatibility for these modules, which means their parameters should be maintained with stable meanings.
+2. If a module or plugin matures, we will remove the 'preview' mark in the documentation. We support (though we cannot guarantee) backwards compatibility for these modules and plugins, which means their parameters should be maintained with stable meanings.
 
-3. If a module's target API changes radically, or if someone creates a better implementation of its functionality, we may mark it deprecated. Modules that are deprecated are still available but they are reaching the end of their life cycle. We retain deprecated modules for 4 release cycles with deprecation warnings to help users update playbooks and roles that use them.
+3. If a module's or plugin's target API changes radically, or if someone creates a better implementation of its functionality, we may mark it deprecated. Modules and plugins that are deprecated are still available but they are reaching the end of their life cycle. We retain deprecated modules and plugins for 4 release cycles with deprecation warnings to help users update playbooks and roles that use them.
```

Comment:
```suggestion
3. If a module's or plugin's target API changes radically, or if someone creates a better implementation of its functionality, it can be marked as deprecated. Modules and plugins that are deprecated are still available but are reaching the end of their lifecycle. Deprecated modules and plugins for 4 release cycles with deprecation warnings to help users update playbooks and roles that use them.
```

Reply:
I think there's an `are retained` missing in the last sentence.

Reply:
Here too, our [style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#stylistic-cheat-sheet) calls for active voice. This change would introduce passive voice. Do you have a strong preference @Akasurde? If so, what's your thinking?

Reply:
I don't have a strong opinion on passive voice.